python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -1449,6 +1472,8 @@ def to_local(self):
         self.accelerated_model = None
         # str indicates current accelerate method
         self.accelerate_method = None
+        self.cxt_manager = DummyForecasterContextManager()
```

Comment:
we should not make this always dummy easily right? since if the model is bf16, we need the forecastercontextmanager.

Reply:
> we should not make this always dummy easily right? since if the model is bf16, we need the forecastercontextmanager.

Well, only when a distributed forecaster is transformed to a local one, the attri is set to be dummy, not always.